salt/grains/core.py

```diff
@@ -494,6 +561,31 @@ def _sunos_memdata():
     return grains
 
 
+def _aix_memdata():
+    '''
+    Return the memory information for AIX systems
+    '''
+    grains = {'mem_total': 0, 'swap_total': 0}
+
```

Comment:
Please remove this empty line.

Reply:
Following style used for _sunos_memdata, _bsd_memdata, etc. already used in the file.

Reply:
changed it anyway